DigitalLearningSolutions.Web/Views/Login/ChooseACentre.cshtml

```diff
@@ -44,17 +44,18 @@
               <span class="nhsuk-table-responsive__heading">Roles: </span>
               @if (centreRow.IsAdmin)
               {
-                <strong class="nhsuk-tag nhsuk-tag--blue centre-role-tag">Admin</strong>
+                <strong class="nhsuk-tag nhsuk-tag--blue centre-role-tag" aria-hidden="true">Admin</strong>
               }

```

Comment:
This approach doesn't really solve the problem in the correct way. We should be trying to achieve this by fixing the html structure rather than hiding things from the screen reader.

The reason these are being read as a single string is because the strong elements doesn't cause a break in content, it's for highlighting content as bold typically. The same is true for span elements.

The correct solution to this would be to either add a div surrounding the strong element like the tags on the expanders in the search/filter pages as a div will cause a pause or to add a visually hidden comma in a span between the two strong elements.
The way this is done for the tags on the various searchable cards is to put a div around the

Reply:
After our conversation, Stella said it was preferred to have the tags read separately, so I went for the divs. That "blank" issue seems to have fixed itself somehow after my laptop crashed and I rebooted 🤷‍♂️

Reply:
Strange, we know it should work OK from the card tags, so I guess we'll just assume it is a one off.